tests/test_options.py

```diff
@@ -8,25 +8,34 @@
 from sphinx.util.images import get_image_size
 from sphinxcontrib.drawio import DrawIOError
 
+def image_size_equal(a, b, blur: int = 2) -> bool:
```

Comment:
> On a purely technical level: this function is not comparing equality, it is comparing rough equality.
> 
> Furthermore, I do not understand the code - why is there a "blur" argument?

I use the word blur because it's a term in image processing. This should not be a very good variable name, I have modified it.

> 
> Something which much more explicitly deals with the fact it's an (height, width) (`width, height`? not sure), rather than tuple indices would be clearer. e.g.

Yes， I add a type hint for the a & b variables too.
> 
> ```python
> max_diff = 2
> ((a_height, a_width), (b_height, b_width)) = (a, b)
> return abs(a_height - b_height) < max_diff and abs(a_width - b_width) < max_diff
> ```

OK, changed to this.